include/antworld/route_continuous.h

```diff
@@ -80,7 +80,7 @@ class RouteContinuous
     std::tuple<meter_t, meter_t> getDistanceToRoute(const Vector2<meter_t> &position) const;
     Pose2<meter_t, degree_t> getPose(meter_t distance) const;
 
-    void setWaypointFamiliarity(size_t pos, double familiarity);
+    void setWaypointFamiliarity(size_t pos, double familiarity) const;
```

Comment:
Again, this makes no sense semantically. ``setWaypointFamilarity`` modifies buffers owned by this object via OpenGL C methods which the compiler doesn't understand but should only be done with a non-const reference to the route

Reply:
You can explicitly suppress ``clang-tidy`` in these places. How would you feel about a ``BOB_NON_CONST`` macro to tag these kinds of methods with?